content/docs/sidebar.json

```diff
@@ -142,6 +142,11 @@
           "checkpoints"
         ]
       },
+      {
+        "label": "Pipelines",
+        "slug": "pipelines",
+        "source": "pipelines/index.md"
```

Comment:
Let's call it "Data Pipelines" or probably even better "ML Pipelines" instead of plain "pipelines".